src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -1889,6 +1905,7 @@ internal class PermissionTest : DatabaseTest() {
         readCohort: Boolean = false,
         readInternalTags: Boolean = false,
         readParticipant: Boolean = false,
+        readSubmission: Boolean = false,
```

Comment:
@sgrimm is there a better place for this? The permission for this is technically allowed for global roles _and_ org members. So it is both globally scoped and organizationally scoped.

Reply:
It might be cleaner to define a new `expect()` method that takes a list of submission IDs. The user with the global role can read all of them, but the org user can only read the ones from their org.

Reply:
I broke it out into its own `expect`.